parts/linux/cloud-init/artifacts/kubelet.service

```diff
@@ -8,6 +8,9 @@ Requires=kms.service
 [Service]
 Restart=always
 EnvironmentFile=/etc/default/kubelet
+{{- if ShouldConfigureHTTPProxy}}
+EnvironmentFile=/etc/environment
+{{- end}}
```

Comment:
remind me: is any difference between /etc/environment vs /etc/default/kubelet for kubelet to take the proxy settings? Are we doing this for other unknown services ?

Reply:
There's no difference, service can have multiple env files. Putting in /etc/environment just so it can be reused. 
Kubelet is the only one for now needs special handling, other services should take from /etc/systemd/system.conf.d/proxy.conf

Reply:
got you. thanks.